src/server/app/util/services/util.server.service.js

```diff
@@ -176,7 +176,7 @@ module.exports.getPage = function (queryParams) {
  * Extract given field from request header
  */
 module.exports.getHeaderField = function (header, fieldName) {
-	return (null == header || null == header[fieldName]) ? null : header[fieldName];
+	return (null == header || null == header[fieldName]) ? null : decodeURI(header[fieldName]);
```

Comment:
Yeah, there was a request to see which client route the user was accessing for auditing.